### PR TITLE
feat(create-collector-step4): apply collector create api

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [x] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA + small changes

Please review order by commits.

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/collector-create/modules/CreateCollectorStep4.vue [hunk 1]
```diff
@@ -84,16 +93,37 @@ const handleClickPrevButton = () => {
...
+                    service_accounts: convertAttachedServiceAccountToIds(),
+                },
+                options: collectorFormState.options,
+                upgrade_mode: collectorFormState.autoUpgrade ? 'AUTO' : 'MANUAL',
+            },
+            schedule: {
+                state: collectorFormState.schedulePower ? 'ENABLED' : 'DISABLED',
+                hours: collectorFormState.scheduleHours,
+            },
+            tags: collectorFormState.tags,
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/collector-create/modules/CreateCollectorStep4.vue:119]: praise: pretty complex!

**Review by yuda110 — APPROVED** [review]
LGTM!!!

**Review by skdud4659 — APPROVED** [review]
It was easy to understand! LGTM!!

File: apps/web/src/services/asset-inventory/collector/collector-create/modules/CreateCollectorStep4.vue [hunk 2]
```diff
@@ -84,16 +93,37 @@ const handleClickPrevButton = () => {
...
+        });
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/collector-create/modules/CreateCollectorStep4.vue:120]: suggestion: What about to use `CollectorCreateParameter` type?

```typescript
const params: CollectorCreateParameter = { ... }
await SpaceConnector.client.inventory.collector.create(params)
```

**Review by WANZARGEN — COMMENTED** [review]
I love the scope of each commit!
They are small enough and separated well so it was very easy to follow your code. :)
I left some comments~!

**Pull request merged**